xc/xenops_server_xen.ml

```diff
@@ -3030,7 +3030,7 @@ module Actions = struct
   let domain_disappeared xc xs domid =
     let token = watch_token domid in
     List.iter (fun d ->
-        List.iter (Xenstore_watch.unwatch ~xs token) (watches_of_device d)
+        List.iter (Xs_watch.unwatch ~xs token) (watches_of_device d)
```

Comment:
Does the name `Xenstore_watch` clash with anything or you just wanted to make sure its using the new code by renaming the module?

Reply:
I could retain the name. This is a result from earlier exploration where there was a problem of having the same module name in ezxenstore and locally. Now the module in ezxenstore is Ez_xenstore_watch and there would be no problem.

So I'll use the `include` you suggested and keep the old name.